etcdserver/api/membership/cluster.go

```diff
@@ -776,3 +785,9 @@ func (c *RaftCluster) VotingMemberIDs() []types.ID {
 	sort.Sort(types.IDSlice(ids))
 	return ids
 }
+
+func (c *RaftCluster) IsAddedToCluster() bool {
+	c.Lock()
+	defer c.Unlock()
+	return c.localNodeAdded
```

Comment:
An alternative to addomg the boolean field would be to check if `c.members[m.localID]` is an entry in the map.

Reply:
Local member could be in the `c.members[]` during initial stage of bootstrapping. But we need to know if `c.members[m.localID]` has been 'updated' by 'AddMember' call, which means the raft log of adding this node is applied locally.